pages.ca/linux/apt-get.md

```diff
@@ -0,0 +1,33 @@
+# apt-get
```

Comment:
`apt-get autoclean` example is missing here too.

Reply:
solved